src/main/webapp/index.html

```diff
@@ -1,13 +1,15 @@
 <!DOCTYPE html>
 <meta charset="UTF-8">
 <title>Open Sesame</title>
+
+<!-- StyleSheets -->
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 <link rel="stylesheet" href="style.css">
 
 <div id="navbar-container"></div>
 <h1 class="display-1 mb-5 mt-5">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
 <spacer class="space"></spacer>
-<button class="btn mt-5 text-center"><span class="display-4">Mentorship Awaits</span></button>
+<button id="login-btn" class="btn mt-5 text-center"><span class="display-4">Mentorship Awaits</span></button>
```

Comment:
I'm thinking we'd like to have an independent sign-up page that this redirects to, but for now this works.